crates/ra_syntax/tests/data/parser/err/0003_C++_semicolon.txt

```diff
@@ -35,5 +35,4 @@ SOURCE_FILE@[0; 40)
       R_CURLY@[38; 39) "}"
   ERROR@[39; 40)
     SEMI@[39; 40) ";"
-error 39: expected item, found `;`
-consider removing this semicolon
```

Comment:
In this case, this error is correct: on the top-level, semicolon is forbidden:

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=f80495a0fafa572b9b5f44c3fec8fc66

So we should still issue this specialized error message, but only if the item is on the top level.

Reply:
Ok, I'll fix this.